open-ce/build_tree.py

```diff
@@ -106,28 +106,30 @@ def _create_recipes(repository, recipes, variant_config_files, variants, channel
     feedstock_conda_build_config_file = build_feedstock.get_conda_build_config()
     if feedstock_conda_build_config_file:
         combined_config_files.append(feedstock_conda_build_config_file)
-    outputs = []
+    build_commands = []
     for recipe in config_data.get('recipes', []):
         if recipes and not recipe.get('name') in recipes:
             continue
         packages, run_deps, host_deps, build_deps, test_deps = _get_package_dependencies(recipe.get('path'),
                                                                                          combined_config_files,
                                                                                          variants)
-        outputs.append(BuildCommand(recipe=recipe.get('name', None),
-                                    repository=repository,
-                                    packages=packages,
-                                    python=variants['python'],
-                                    build_type=variants['build_type'],
-                                    mpi_type=variants['mpi_type'],
-                                    cudatoolkit=variants['cudatoolkit'],
-                                    run_dependencies=run_deps,
-                                    host_dependencies=host_deps,
-                                    build_dependencies=build_deps,
-                                    test_dependencies=test_deps,
-                                    channels=channels if channels else []))
+        build_commands.append(BuildCommand(recipe=recipe.get('name', None),
+                                           repository=repository,
+                                           packages=packages,
+                                           python=variants['python'],
+                                           build_type=variants['build_type'],
+                                           mpi_type=variants['mpi_type'],
+                                           cudatoolkit=variants['cudatoolkit'],
+                                           run_dependencies=run_deps,
+                                           host_dependencies=host_deps,
+                                           build_dependencies=build_deps,
+                                           test_dependencies=test_deps,
+                                           channels=channels if channels else []))
+
+    test_commands = test_feedstock.gen_test_commands()
```

Comment:
gen_test_commands() has a argument which is default to tests/open-ce-tests.yaml. I don't see that file anywhere in the repo. Is gen_test_commands() independent of any of the arguments of _create_commands(...)? If yes, then it is generic for all and should not be here. I think I am missing something here.

Reply:
Now I got where is that open-ce-tests.yaml file. It will be added to each feedstock.